src/main/java/de/tum/in/www1/artemis/service/feature/FeatureToggleService.java

```diff
@@ -95,6 +95,6 @@ public boolean isFeatureEnabled(Feature feature) {
      * @return A list of disabled features
      */
     public List<Feature> disabledFeatures() {
-        return features.entrySet().stream().filter(e -> Boolean.FALSE.equals(e.getValue())).map(Map.Entry::getKey).collect(Collectors.toList());
+        return features.entrySet().stream().filter(feature -> Boolean.FALSE.equals(feature.getValue())).map(Map.Entry::getKey).collect(Collectors.toList());
```

Comment:
```suggestion
        return features.entrySet().stream().filter(feature -> Boolean.FALSE.equals(feature.getValue())).map(Map.Entry::getKey).toList();
```
The `Collectors` import is going to be unused afterwards.